UIPS/UIP-0131.md

```diff
@@ -0,0 +1,121 @@
+---
+uip: "0131"
+title: "Ministry"
+description: An Azimuth Multibridge
+author: ~master-malwyl, ~rolryx, ~sarlev-sarsen
+status: Draft
+type: Standards Track
+category: PKI
+created: ~2024.11.20
+---
+
+Abstract
+--------
+
+Ministry enables Urbit's Azimuth PKI to operate across multiple blockchain platforms while maintaining Ethereum as its root of trust. The system introduces a secure bridging protocol that allows points to move between supported chains through galaxy-operated oracles. This proposal specifies both the general bridge architecture and a concrete first implementation supporting Coinbase's Base layer-2 rollup.
+
+Motivation
+----------
+
+The Urbit namespace currently exists solely on Ethereum. While Ethereum provides strong property rights guarantees, limiting Azimuth to a single chain restricts user optionality and increases operational costs. As the blockchain ecosystem evolves with new Layer 2s and alternative Layer 1s, there is value in making Urbit IDs portable while preserving both the security guarantees and property rights of the existing system.
+
+Moving points to more cost-effective chains reduces operational overhead for users. Integration with multiple blockchain ecosystems increases accessibility and enables future compatibility with specialized Urbit-specific chains or features available on select global consensus ledgers other than mainnet Ethereum. The proposal maintains security through a galaxy-based oracle system with robust economic incentives.
```

Comment:
I think 'accessibility' is probably not the right word here, but rather this is a continuation of the point about optionality (thus enabling 'access' for people not interested in what Ethereum has to offer). Having your Urbit ID legible to the chain on which you do most of your blockchain activity is quite desirable. On Ethereum and EVM related chains this looks like stuff like [Tokenbound Accounts](https://eips.ethereum.org/EIPS/eip-6551) (smart contract wallets controlled by the owner of your L1 Urbit ID NFT). On bitcoin it might look like derived payment addresses, on Zenith it might look like access to the hyperscry oracle, or on Solana something else entirely (not a solana enjoyer, I'll let the degens make that case).